Receiver/Receiver.test.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+import Receiver
+
+
+class ReceiverTest(unittest.TestCase):
```

Comment:
Excellent coverage of positive and negative scenarios. You have tested individual functionality at various levels. Keep it up 👍